proto/dropbox/dropbox.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package dropbox;
+
+service Dropbox {
+  rpc GetApproverInfo(GetApproverInfoRequest) returns(GetApproverInfoResponse) {
+  }
+  rpc CanApproveToUser(CanApproveToUserRequest)
+      returns(CanApproveToUserResponse) {}
+  rpc CreateRequest(CreateRequestRequest) returns(CreateRequestResponse) {}
```

Comment:
`CreateRequestRequest` and `CreateRequestResponse` sounds kind of silly, doesn't it?
I think you should give them (and the function) more meaningful names.